src/main/target/common_post.h

```diff
@@ -426,7 +465,7 @@
 #endif
 
 // Generate USE_SPI_GYRO or USE_I2C_GYRO
-#if defined(USE_GYRO_L3G4200D) || defined(USE_GYRO_MPU3050) || defined(USE_GYRO_MPU6000) || defined(USE_GYRO_MPU6050) || defined(USE_GYRO_MPU6500)
+#if defined(USE_GYRO_MPU6000) || defined(USE_GYRO_MPU6050) || defined(USE_GYRO_MPU6500)
```

Comment:
This should be removed.

Reply:
This is needed to apply for I2C gyros ?

Reply:
My point is it should be specified by the config file not added here.

Reply:
@blckmn fixed in last commit.